.gitignore

```diff
@@ -34,8 +34,17 @@ build/Release
 node_modules
 jspm_packages
 
+# Npm lock files
+package-lock.json
+
 # Optional npm cache directory
 .npm
 
 # Optional REPL history
 .node_repl_history
+
+# Temp directories
+TEMP
+
+# Resources
+Strings/*
```

Comment:
We need these files.

Reply:
Thought I removed that one >..<

Reply:
Does TEMP need to be removed from the .gitignore as well?